### PR TITLE
fix #76928 ustream.to

### LINKED ISSUE #76928: UStream.to Adblock Detected
When opening any channel on UStream it will detect Ublock Origin and it won't work until it's disabled.

Upon disabling the page will be filled with redirects, popups and ads

Checking the logger, UStream will change the key for the stream.

For example:
https://www.ustream.to/live?id=sic-portugale&program=2b44bd0096f425813299c2b1539b90cc

UBlock Disabled:
https://hls.ustream.to/SIC-Portugal.m3u8?token=dec-4f3-99d-e6c-bce-aef-c6f-c94-c5e-ab7-1d9-b0e-d2c-853-5f6-70f-f78-797-256-eef-85c-9-a5d-d1c-e8t-f6c

UBlock Enabled:
https://hls.ustream.to/SIC-Portugal.m3u8?token=adblocker-a5d-d1c-e8t-f6c

uBlock Origin solved it by doing the following:

*$script,redirect-rule=noopjs,domain=beef1999.blogspot.com
@@||bestdisplayformats.com^$script,domain=beef1999.blogspot.com
beef1999.blogspot.com##+js(aopr, console.clear)
beef1999.blogspot.com##+js(aopr, LieDetector)
beef1999.blogspot.com##.popup

### PR DESCRIPTION
**Pull request by wout-codes**
#76928
This rule `ustream.to#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "constructor")` is to prevent the site from calling useless ad-servers. And this rule `#%#//scriptlet("prevent-addEventListener", "", "[$.")` is to prevent further popups.
The best way to prevent/dismiss anti-adblock is to use the `prevent-eval-if ` scriptlet.

### CODE REVIEW

**Comment by wout-codes** [comment]
@Alex-302 @AdamWr @zloyden I request your for your review.

**Comment by wout-codes** [comment]
I think that's all. @Alex-302 @AdamWr @zloyden I want your reviews.

**Comment by Alex-302** [comment]
`beef1999.blogspot.com` - wrong domain in the rules?

**Comment by wout-codes** [comment]
> `beef1999.blogspot.com` - wrong domain in the rules?

Haha. @Alex-302 You're funny. Most of the anti-adblocks and popups appears from `beef1999.blogspot.com`. Only one or two from `ustream.to`.

**Comment by Alex-302** [comment]
Hm. Did not see that request.

**Comment by wout-codes** [comment]
@Alex-302 If you are unsure, you can just rely on me, I would not add redundant rules :)
<details>

image
</details>

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1,6 +1,12 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76928
+beef1999.blogspot.com#?##banner_app
+beef1999.blogspot.com#?#.popup
+beef1999.blogspot.com#?##boz_av_bottom
+! https://github.com/AdguardTeam/AdguardFilters/issues/77063
```
Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt:8]: What is it?

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:8]: What? Why is it added? Lemme remove it.

Comment (Alex-302) [reply EnglishFilter/sections/css_extended.txt]: Bad merge. This rule was removed
https://github.com/AdguardTeam/AdguardFilters/commit/695921d4ba6543c4345bbc2c096880397f85ad07

**Review by Alex-302 — APPROVED** [review]

**Comment by wout-codes** [comment]
@Alex-302 It's a social leftover. Should be fixed in social filter ext css. Otherwise there will be False positive.
image

**Comment by zloyden** [comment]
This should work `ustream.to##.container > p[style^="color:"] + hr + div[onclick^="window.open("]` or try `ustream.to#?#.container > div[onclick] > a.twitter-share-button:upward(1)`

**Comment by wout-codes** [comment]
> [#77108 (comment)](https://github.com/AdguardTeam/AdguardFilters/pull/77108#issuecomment-795574042) This should work `ustream.to##.container > p[style^="color:"] + hr + div[onclick^="window.open("]` or try `ustream.to#?#.container > div[onclick] > a.twitter-share-button:upward(1)`

Nice magic. Can you add directly? Please

**Comment by wout-codes** [comment]
> [#77108 (comment)](https://github.com/AdguardTeam/AdguardFilters/pull/77108#issuecomment-795574042) This should work `ustream.to##.container > p[style^="color:"] + hr + div[onclick^="window.open("]` or try `ustream.to#?#.container > div[onclick] > a.twitter-share-button:upward(1)`

You're a magician so you can do anything. https://github.com/AdguardTeam/AdguardFilters/pull/77208

**Pull request merged**